docs/_src/api/api/utils.md

```diff
@@ -0,0 +1,301 @@
+<a id="export_utils"></a>
+
+# Module export\_utils
+
+<a id="export_utils.print_answers"></a>
+
+#### print\_answers
+
+```python
+def print_answers(results: dict, details: str = "all", max_text_len: Optional[int] = None)
+```
+
+Utility function to print results of Haystack pipelines
+
+**Arguments**:
+
+- `results`: Results from a pipeline
+- `details`: One of "minimum", "medium", "all". Defining the level of details to print.
+- `max_text_len`: shorten lengthy text fields to the maximum allowed length. Set to
+None to not cut long text.
+
+**Returns**:
+
+None
+
+<a id="export_utils.print_documents"></a>
+
+#### print\_documents
+
+```python
+def print_documents(results: dict, max_text_len: Optional[int] = None, print_name: bool = True, print_meta: bool = False)
+```
+
+Utility that prints a compressed representation of the documents returned by a pipeline.
+
+**Arguments**:
+
+- `max_text_lenght`: shorten the document's content to a maximum number of chars. if None, does not cut.
+- `print_name`: whether to print the document's name (from the metadata) or not.
+- `print_meta`: whether to print the document's metadata or not.
+
+<a id="export_utils.print_questions"></a>
+
+#### print\_questions
+
+```python
+def print_questions(results: dict)
+```
+
+Utility to print the output of a question generating pipeline in a readable format.
+
+<a id="export_utils.export_answers_to_csv"></a>
+
+#### export\_answers\_to\_csv
+
+```python
+def export_answers_to_csv(agg_results: list, output_file)
+```
+
+Exports answers coming from finder.get_answers() to a CSV file
+
+**Arguments**:
+
+- `agg_results`: list of predictions coming from finder.get_answers()
+- `output_file`: filename of output file
+
+**Returns**:
+
+None
+
+<a id="export_utils.convert_labels_to_squad"></a>
+
+#### convert\_labels\_to\_squad
+
+```python
+def convert_labels_to_squad(labels_file: str)
+```
+
+Convert the export from the labeling UI to SQuAD format for training.
+
+**Arguments**:
+
+- `labels_file`: path for export file from the labeling tool
```

Comment:
Does it have to be the file exported from the labelling tool or can it be any file with labels?

I'd change to: The path to the file containing labels.